src/app/api/(auth)/register/route.ts

```diff
@@ -0,0 +1,3 @@
+export async function GET(request: Request) {
+	return new Response('Hello from NextJS!');
+}
```

Comment:
I guess this will be updated later by someone? Maybe have a comment in the code here that says that.

Reply:
Sure thing! Fixed ✅